### PR TITLE
New html term label

### PR DESCRIPTION
**Pull request by Armania**
### Description

Adding a new entry for HTML term "label" tag. I tried to copy as much of the formatting from the input definition but I will need help reviewing this and making sure the style is matched. This is my first draft.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [ x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/elements/terms/label/label.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Label
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:1]: The metadata for this entry is missing. You can look at the source for one of the other HTML entries to see what it looks like.  Also review the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md).

File: content/html/concepts/elements/terms/label/label.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+The <label> tag defines a label for several elements:
+
+- [**`<input>`**](../input/input.md)
+- [**`<meter>`**](../meter/meter.md)
+- [**`<progress>`**](../progressput/progress.md)
+- [**`<select>`**](../select/select.md)
+- [**`<textarea>`**](../textarea/textarea.md)
+
+**Note:** Labels are important for accessbility. They allow screenreaders to connect this caption to its coresponding ui elements.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:21]:
```suggestion
> **Note:** Labels are important for accessibility. They allow screenreaders to connect this caption to its corresponding UI elements.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Syntax
+
+```html
+<label>Input Label</label>
+```
+
+## Usages
+
+The <label> tag defines a label for several elements:
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:13]:
```suggestion
The <label> tag can define a label for several elements:
```

File: content/html/concepts/elements/terms/label/label.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+|   `form`    |   String   | Specifies which form the label belongs to.
+
+```html
+<form id="contact">
+  <label form="contact" for="name">Input Label</label>
+  <input id="name" />
+</form>
+```
+
+**Note:** UI elements can be connected to any **`<label>`** element when placed between **`<label></label>`** tags.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:39]:
```suggestion
> **Note:** UI elements can be connected to any `<label>` element when placed between `<label></label>` tags.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+The <label> tag defines a label for several elements:
+
+- [**`<input>`**](../input/input.md)
+- [**`<meter>`**](../meter/meter.md)
+- [**`<progress>`**](../progressput/progress.md)
+- [**`<select>`**](../select/select.md)
+- [**`<textarea>`**](../textarea/textarea.md)
+
+**Note:** Labels are important for accessbility. They allow screenreaders to connect this caption to its coresponding ui elements.
+Also, when a user clicks the text within the **`<label>`** element, it toggles the coresponding input. This is great for users who have difficulty clicking on very small ui elements such as checkboxes.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:22]:
```suggestion
Also, when a user clicks the text within the `<label>` element, it toggles the corresponding input. This is great for users who have difficulty clicking on very small UI elements such as checkboxes.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+Multiple label elements can be used for the same ui element.
+
+```html
+<label for="name">Name Input Label
+<input id="name" />
+<label for="name">Warning: Name is required</label>
+```
+
+**warning:** Do not place anchors, buttons, or any interactive ui element, between  **`<label></label>`** tags.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:58]:
```suggestion
> **Note:** Do not place anchors, buttons, or any interactive UI element, between  `<label></label>` tags.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+## Example
+
+The following examples uses a `<form>` element that features `<input>` elements with coresponding `<label>` tags:
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:62]:
```suggestion
The following examples uses a `<form>` element that features `<input>` elements with corresponding `<label>` tags:
```

File: content/html/concepts/elements/terms/label/label.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+```html
+<label for="name">
+  Name Input Label
+  <input id="name" />
+</label>
+```
+
+## Multiple labels
+
+Multiple label elements can be used for the same ui element.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:50]:
```suggestion
Multiple label elements can be used for the same UI element.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Armania. I did a first review and left some comments. The most important thing is to add the missing Metadata to the top of the file.

File: content/html/concepts/elements/terms/label/label.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+  - 'Label'
+  - 'Interface'
+  - 'Tags'
+  - 'UI'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [**`<input>`**](../input/input.md) element. The <label> tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:19]:
```suggestion
The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+## Syntax
+
+```html
+<label>Input Label</label>
+```
+
+## Usages
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:29]: Remove the suggestion block, just leave the sentence.

File: content/html/concepts/elements/terms/label/label.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+The <label> tag can define a label for several elements:
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:30]:
```suggestion
The `<label>` tag can define a label for several elements:
```

File: content/html/concepts/elements/terms/label/label.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+```
+
+- [**`<input>`**](../input/input.md)
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:33]: Remove the bold on these.

File: content/html/concepts/elements/terms/label/label.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+- [**`<meter>`**](../meter/meter.md)
+- [**`<progress>`**](../progressput/progress.md)
+- [**`<select>`**](../select/select.md)
+- [**`<textarea>`**](../textarea/textarea.md)
+
+```suggestion
+> **Note:** `<label>` tags are essential for good accessbility.
+>
+> - They allow screenreaders to connect this caption to its coresponding ui elements.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:42]:
```suggestion
> - They allow screen readers to connect this caption to its corresponding UI elements.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+> - Also, when a user clicks the text within the **`<label>`** element, it toggles the coresponding input. This is great for users who have difficulty clicking on very small ui elements such as checkboxes.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:43]:
```suggestion
> - Also, when a user clicks the text within the `<label>` element, it toggles the corresponding input. This is great for users who have difficulty clicking on very small UI elements such as checkboxes.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+
+## Atributes
+
+Some common attributes used to set properties for the `<label>` element are shown below:
+
+|   Attribute    |   Data Type   | Description
+|   `for`    |   String   | Specifies the id of the form element the `<label>` should be bound to.
+|   `form`    |   String   | Specifies which form the `<label>` belongs to.
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:54]: remove the suggestion block and just leave the sentence.

File: content/html/concepts/elements/terms/label/label.md [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+```
+
+```html
+<form id="contact">
+  <label form="contact" for="name">Input Label</label>
+  <input id="name" />
+</form>
+```
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:65]: Remove the suggestion block and just leave the sentence.

File: content/html/concepts/elements/terms/label/label.md [hunk 17]
```diff
@@ -0,0 +1,133 @@
...
+
+Multiple `<label>` elements can be used for the same ui element.
+
+```html
+<label for="name">Name Input Label
+<input id="name" />
+<label for="name">Warning: Name is required</label>
+```
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:90]: Remove the suggestion block and just leave the sentence.

File: content/html/concepts/elements/terms/label/label.md [hunk 18]
```diff
@@ -0,0 +1,133 @@
...
+> **Note:** Do not place anchors, buttons, or any interactive UI element, between  `<label></label>` tags.
+```
+
+## Example
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:96]: Remove the suggestion block and just leave the sentence.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Armania, I left several more comments. One thing that needs to be fixed: The suggestion block surrounding the various suggested edits is not part of the edit. It's only marking the text that needs to be changed. The opening \`\`\`suggestion and closing \`\`\` should not be included in the edit.

**Comment by Armania** [comment]
Wanted to check-in and see if I left anything undone

File: content/html/concepts/elements/terms/label/label.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+  - 'Input'
+  - 'Interface'
+  - 'Tags'
+  - 'UI'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<label/>` element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [**`<input>`**](https://www.codecademy.com/resources/docs/html/elements/input/input.md) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:20]:
```suggestion
The **`<label/>`** element is used to identify captions for other HTML elements. This is most commonly used with form elements such as the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input/input.md) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
```

File: content/html/concepts/elements/terms/label/label.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+The `<label>` tag can define a label for several elements:
+
+- [`<input>`](../input/input.md)
+- [`<meter>`](../meter/meter.md)
+- [`<progress>`](../progressput/progress.md)
+- [`<select>`](../select/select.md)
+- [`<textarea>`](../textarea/textarea.md)
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:38]: You still need to remove ```suggestion blocks around text.

File: content/html/concepts/elements/terms/label/label.md (hunk 10 above)
Comment (SSwiniarski) [reply content/html/concepts/elements/terms/label/label.md:38]: Still needs to be done

File: content/html/concepts/elements/terms/label/label.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+> **Note:** `<label>` tags are essential for good accessbility.
+```
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:42]: Remove ```suggestion blocks around text

File: content/html/concepts/elements/terms/label/label.md [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+> - '<label>' tags creates captions for its coresponding UI elements.
+```
+
+```suggestion
```
Comment (SSwiniarski) [inline content/html/concepts/elements/terms/label/label.md:46]: Remove ```suggestion blocks around text.

File: content/html/concepts/elements/terms/label/label.md (hunk 7 above)
Comment (SSwiniarski) [reply content/html/concepts/elements/terms/label/label.md:93]: This still needs to be done.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Armania there are still a number of edits to be done, primarily removing the ```suggestion blocks around text. You marked these resolved without making any edits. They still need to be removed.

**Comment by yangc95** [comment]
@Armania did this branch get rebased? I think I'd be best to create another PR off the `main` branch so all the authors aren't an issue for the CLA check. Thanks in advance!

**Comment by Armania** [comment]
@yangc95 + @SSwiniarski , please see https://github.com/Codecademy/docs/pull/1429 for my new pull request

**Comment by SSwiniarski** [comment]
Closing this in favor of [1429](https://github.com/Codecademy/docs/pull/1429).

**Pull request closed without merging**